MetabolicAtlas/MetabolicAtlas
# feat: got enzymes range filters

**Related issue(s) and PR(s)**  
This PR closes #1026.



**Type of change**  

 
**List of changes made**  

- Add a range filter component an use it as a custom filter for `kcat` values in the enzymes table.

**Screenshot of the fix**  

![image](https://user-images.githubusercontent.com/423498/174820908-fd876528-2dc5-4a87-b33e-67808afc04cb.png)
![image](https://user-images.githubusercontent.com/423498/174820422-b808ef4d-8242-471a-9ada-e03ab342c2c3.png)

**Testing**  

- Relative urls that can be reused both for production and local testing
- Instructions on how to test
1. Go to a details page, for example `/gotenzymes/ec/2.5.1.19`.
2. Try different values for the `min` and `max` inputs for the `kcat` column.
3. Verify that the table is updated accordingly. This should also work in combination with other filters and column sorting.

**Further comments**  


**Definition of Done checklist**
